app/views/pages/governance/bida.njk

```diff
@@ -78,7 +82,7 @@
                   <div class="nhsuk-promo__content">
                     <h3 class="nhsuk-promo__heading">The EAB and its sub-group’ decisions are updated by the secretariat here</h3>
                     <p class="nhsuk-promo__description">
-                      All queries regarding the decisions should be directed to <a href="mailto:ea-da@nhs.net">ea-da@nhs.net</a>
```

Comment:
These a mailto: links have been removed as they break the layout of the link box.